protocol/x/delaymsg/keeper/dispatch_test.go

```diff
@@ -53,7 +52,8 @@ func TestDispatchMessagesForBlock(t *testing.T) {
 	require.Equal(t, []uint32{0, 1, 2}, blockMessageIds.Ids)
 
 	// Mock the bridge keeper methods called by the bridge msg server.
-	bridgeKeeper.On("CompleteBridge", ctx, mock.Anything).Return(nil).Times(len(constants.AllMsgs))
+	bridgeKeeper.On("CompleteBridge", mock.AnythingOfType("types.Context"), mock.Anything).
```

Comment:
Do we have a test somewhere in this file that tests of panic (e.g. `CompleteBridge`) panics?

Reply:
No, but there is a unit tests for RunCached that validates that panics are not propagated.

Reply:
Added one just in case!